### PR TITLE
ISPN-13981: Restore the Enabling JMX remote ports module on the 13.0.x branch

### PR DESCRIPTION
**Pull request by dfitzmau**
JIRA: https://issues.redhat.com/browse/ISPN-13981

### CODE REVIEW

**Review by dvagnero — CHANGES_REQUESTED** [review]
@dfitzmau The file is missing conditionals and introduces several blank lines. Please copy the file from main

**Comment by dfitzmau** [comment]
Hello @tristantarrant . Would you be OK to review this PR for the restored file?

File: documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[id='enabling-jmx-port_{context}']
+= Enabling JMX remote ports
+Provide unique remote JMX ports to expose {brandname} MBeans through connections in JMXServiceURL format.
+
+// Server Guide
+ifdef::remote_caches[]
+[NOTE]
+====
+{brandname} Server does not expose JMX remotely by using the single port endpoint. If you want to remotely access the {brandname} Server through JMX, you must enable a remote port.
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc:9]: Each sentence should start on its own line
```suggestion
{brandname} Server does not expose JMX remotely by using the single port endpoint.
If you want to remotely access the {brandname} Server through JMX, you must enable a remote port.
```

Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc]: Thanks, @dvagnero .

I updated the content based on your feedback.

**Review by dvagnero — APPROVED** [review]
LGTM! Thanks for the improvements

**Comment by tristantarrant** [comment]
Suggestion: we could add a link to the Java docs that show how to configure security https://docs.oracle.com/en/java/javase/11/management/monitoring-and-management-using-jmx-technology.html#GUID-F08985BB-629A-4FBF-A0CB-8762DF7590E0

**Comment by dfitzmau** [comment]
> 

Thanks, @tristantarrant .

I provided the link in the context of the upstream docs. Downstream might be trickier, so I added an if-def statement. I hope this is OK?

**Pull request merged**